addons/tls13-comparison-v1/bootstrap.js

```diff
@@ -0,0 +1,90 @@
+/* -*- indent-tabs-mode: nil; js-indent-level: 2 -*- */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+
+let {classes: Cc, interfaces: Ci, utils: Cu} = Components;
+
+Cu.import("resource://gre/modules/Preferences.jsm");
+Cu.import("resource://gre/modules/Services.jsm");
+Cu.import("resource://gre/modules/TelemetryController.jsm");
+Cu.import("resource://gre/modules/ClientID.jsm");
+
+Cu.importGlobalProperties(["crypto", "TextEncoder"]);
+const DEBUG = false;
+const ENABLE_PROB = .5;
+let gStarted = false;
+const kVERSION_MAX_PREF = "security.tls.version.max";
+
+function debug(msg) {
+    if (DEBUG) {
+        console.log(`TLS 1.3 Test: ${msg}`);
+    }
+}
+
+function disable() {
+  Experiments.instance().disableExperiment("FROM_API");
```

Comment:
Also not sure if it's necessary to have a `disable` function if it only contains this one line, might as well do the `disableExperiment` one-liner.